### PR TITLE
34 friends request delete search page

### LINKED ISSUE #65: Create Layout for RN App
Similar to this:

Image

### PR DESCRIPTION
**Pull request by AriA700**
Includes the friends request page and layout

### CODE REVIEW

File: package-lock.json
Comment (mattcab2002) [inline package-lock.json:1]: I think this one was comited by mistake

File: .gitignore
Comment (mattcab2002) [inline .gitignore:1]: No need for this because there's one in the frontend folder that handles node_modules already

File: .vscode/settings.json
Comment (mattcab2002) [inline .vscode/settings.json:1]: Just remove this one if u can

Comment (mattcab2002) [inline .vscode/settings.json:1]: Just remove this one if u can

**Comment by mattcab2002** [comment]
Merge with frontend to solve the conflicts in this PR and make sure it still runs proper

File: package-lock.json
Comment (AriA700) [reply package-lock.json:1]: Not sure what you mean by this, but the rest of the changes are done

Comment (mattcab2002) [reply package-lock.json:1]: this package-lock.json is outside of the frontend folder

Comment (mattcab2002) [reply package-lock.json:1]: when i checkout to this branch i see that there is a package-lock.json, .vscode, and .gitignore on the same level as the frontend directory

image

Comment (AriA700) [reply package-lock.json:1]: ok yeah, now I remember why these were created. I have now removed them.

**Review by mattcab2002 — APPROVED** [review]
Just need to fix merge conflicts

**Review by Arty2001 — APPROVED** [review]

**Pull request merged**